neos/neos-ui
# TASK: Display number of validation errors for each tab in inspector

**What I did**
Completed issue #2377. The number of errors now gets displayed as a badge at the top right corner. This helps to show, why the "Apply" button is disabled when errors are in other tabs and thus not visible.

**How I did it**
The "validationError" object was already available in the redux store. I used it to summarize a count for each tab by testing how many of the field ids, whose values are faulty, are contained inside a tab group. Finally, I created a notification badge for the "Tabs" component, which is displayed if the count is higher than 0.

**How to verify it**
Delete the value of for instance a mandatory field and check if a badge with the number 1 will appear.

<img width="334" alt="Bildschirmfoto 2019-03-19 um 16 53 20" src="https://user-images.githubusercontent.com/17150659/54621136-8d0c6f00-4a67-11e9-9636-b5f953487ed5.png">
